config/config.default.php

```diff
@@ -278,7 +278,7 @@
     'enable_goody'            => (bool)env('ENABLE_GOODY', false),
 
     // Enables the food voucher in the user profile
-    'enable_voucher'            => (bool)env('ENABLE_VOUCHER', true),
+    'enable_voucher'          => (bool)env('ENABLE_VOUCHER', true),
```

Comment:
Can you try to add a rule that covers this style to the [phpcs rule set](https://github.com/engelsystem/engelsystem/blob/main/.phpcs.xml)?

Reply:
never worked with phpcs. i only aligned it as all the other indentations

Reply:
Never mind. I think this only works for direct siblings.